### ISSUE TITLE
[Accessibility]A11y_VSCode_MenuBar_Keyboard: Using keyboard keys the Menu Bar is not accessible

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#51954]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
Keyboard,MAS2.1.1,MAS2.4.3

**Repro Steps:**
1.Navigate to "Menu Bar" at top of the screen after login.
2.Try to navigate "Menu Bar" through the keyboard key such as "TAB".
3.Observed that,Using tab key "Menu Bar" is not accessible.

**Actual:**
Using only shortcut keys such as "Alt+F,Alt+E,Alt+S" menu bar is accessible but when user want to access the menu bar using keyboard key such as "tab,arrow keys"  then user is not able to access it.

**Expected:**
The "Menu Bar" should be accessible by the shortcut keys as well as "tab and arrow keys".

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/fixit-details.aspx?fixitid=326](url)

**User Impact:**
If user does not know the short cut key and only accessible by the "tab,arrow keys"
then user will not able to access the menu bar.

**MAS Reference:**
MAS2.1.1- [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}](url)

MAS2.4.3- [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}](url)

**Attachment For Reference:**
[[Accessibility]A11y_VSCode_MenuBar_Keyboard.pptx](https://github.com/Microsoft/vscode/files/2104757/Accessibility.A11y_VSCode_MenuBar_Keyboard.pptx)
Does this issue occur when all extensions are disabled?: Yes

**Comment by sbatten** [comment]
Working with new custom menubar
